test/harness.ts

```diff
@@ -362,6 +362,7 @@ expect.extend({
       cmd: [bunExe(), ...cmds],
       env: bunEnv,
       stdio: ["inherit", "pipe", "inherit"],
+      cwd: process.cwd(),
```

Comment:
This shouldn't be necessary, this is the default value. Unless there's code which is overriding the cwd function.